src/raftstore/store/store.rs

```diff
@@ -586,9 +589,10 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             .next() {
             let exist_region = self.region_peers[&exist_region_id].region();
             if enc_start_key(exist_region) < data_end_key(msg.get_end_key()) {
-                debug!("msg {:?} is overlapped with region {:?}, ignored",
+                debug!("msg {:?} is overlapped with region {:?}, queue for later split",
```

Comment:
In my previous test, the new split leader vote messages will be dropped here https://github.com/pingcap/tikv/blob/48d44fdd861a94f7aed0774f85a6645b046f7fbd/src/raftstore/store/store.rs#L580